docs/en/interfaces/http.md

```diff
@@ -133,8 +133,14 @@ You can use the internal ClickHouse compression format when transmitting data. T
 If you specified `compress=1` in the URL, the server compresses the data it sends you.
 If you specified `decompress=1` in the URL, the server decompresses the same data that you pass in the `POST` method.
 
-It is also possible to use standard `gzip`-based [HTTP compression](https://en.wikipedia.org/wiki/HTTP_compression). To send a `POST` request compressed using `gzip`, append the request header `Content-Encoding: gzip`.
-In order for ClickHouse to compress the response using `gzip`, you must append `Accept-Encoding: gzip` to the request headers, and enable the ClickHouse [enable_http_compression](../operations/settings/settings.md#settings-enable_http_compression) setting. You can configure the compression level of the data with the [http_zlib_compression_level](#settings-http_zlib_compression_level) setting.
+It is also possible to use other compression methods for HTTP data. To send a `POST` request compressed, append the request header `Content-Encoding: compression_method`. In order for ClickHouse to compress the response, you must append `Accept-Encoding: compression_method`. You can use the following compression methods:
+
+- Standard `gzip`-based [HTTP compression](https://en.wikipedia.org/wiki/HTTP_compression) (`compression_method = gzip`).
```

Comment:
* No need in backticks around gzip.
* Why wikipedia link is here instead of one of previous paragraphs?